CMakeLists.txt

```diff
@@ -202,21 +187,25 @@ set(SOURCES
     ${PROJECT_SOURCE_DIR}/src/tagpool.h
     ${PROJECT_SOURCE_DIR}/src/tagtreedelegateeditor.cpp
     ${PROJECT_SOURCE_DIR}/src/tagtreedelegateeditor.h
-    ${PROJECT_SOURCE_DIR}/src/theme.cpp
-    ${PROJECT_SOURCE_DIR}/src/theme.h
     ${PROJECT_SOURCE_DIR}/src/trashbuttondelegateeditor.cpp
     ${PROJECT_SOURCE_DIR}/src/trashbuttondelegateeditor.h
     ${PROJECT_SOURCE_DIR}/src/treeviewlogic.cpp
     ${PROJECT_SOURCE_DIR}/src/treeviewlogic.h
+    ${PROJECT_SOURCE_DIR}/src/editorsettingsoptions.h
```

Comment:
Nit: It'd be good to keep this list of files sorted alphabetically...

Reply:
👍